### PR TITLE
Adjust recommended apps page

### PR DESCRIPTION
**Pull request by PVince81**
Removed collabora code.
Added description for richdocuments as the name will be "Nextcloud Office" pulled from the app store.

image

~~Note: currently it still says "Collabora Online" and the text will change as soon as the app store name changes to "Nextcloud Office" as well in the next app release: https://apps.nextcloud.com/apps/richdocuments~~

### CODE REVIEW

**Comment by PVince81** [comment]
wait, now I realize that we only wanted to **remove collabora code** from the list, assuming that it would be installed anyway.

but now I renamed it from the code, so it won't be installed...

I'll bring it back on the backend only...

**Comment by PVince81** [comment]
okay, I pushed again, now, as per requirements: richdocumentscode doesn't appear in the visible list but will still be installed

**Comment by PVince81** [comment]
and, I added a name override for Collabora Office so that it always appears as "Nextcloud Office"

File: core/src/components/setup/RecommendedApps.vue [hunk 1]
```diff
@@ -185,13 +187,25 @@ export default {
...
 				logger.warn(`no app description for recommended app ${appId}`)
 				return ''
 			}
 			return recommended[appId].description
 		},
+		isHidden(appId) {
+			if (!(appId in recommended)) {
+				return false
+			}
+			return !!recommended[appId].hidden
```
Comment (szaimen) [inline core/src/components/setup/RecommendedApps.vue:207]:
```suggestion
			return recommended[appId].hidden
```

Comment (PVince81) [reply core/src/components/setup/RecommendedApps.vue:207]: why ? I just wanted to force it to be bool even when undefined.
might be overkill, yes

Comment (szaimen) [reply core/src/components/setup/RecommendedApps.vue:207]: ah, I see

**Review by szaimen — APPROVED** [review]
LGTM but didn't test

**Comment by szaimen** [comment]
BTW @PVince81 didn't we also put recognize into recommended apps?

**Review by Pytal — APPROVED** [review]

**Comment by PVince81** [comment]
@szaimen see https://github.com/nextcloud/firstrunwizard/pull/739

**Pull request merged**